### PR TITLE
#162725432 Admin activate or deactivate question

### PR DESCRIPTION
**Pull request by michael-basweti**
**What does this PR do?**
- Admin is able to activate and deactivate a question

**How should this be tested?**
- Run the server `http://127.0.0.1:5000/mrm`.
- Run the following query
    ```
   mutation {
      updateQuestion(questionId:1,
      isActive: false) {
        question {
          id,
          isActive
        }
      }
    }
   ```

**What are the relevant pivotal tracker stories?**
[#162240864](https://www.pivotaltracker.com/n/projects/2154921/stories/162725432)

screenshot 2018-12-19 at 12 08 57
screenshot 2018-12-19 at 12 09 25

### CODE REVIEW

**Comment by Bonifase** [comment]
Fix the failing tests and the merge conflict, also you should only have one commit message in your PR

File: alembic/versions/5f9ad1c6faa2_migration_message.py [hunk 1]
```diff
@@ -0,0 +1,189 @@
+"""Migration message
```
Comment (Bonifase) [inline alembic/versions/5f9ad1c6faa2_migration_message.py]: You should generate only one new alembic file for your version upgrade with a more descriptive message

Comment (michael-basweti) [reply alembic/versions/5f9ad1c6faa2_migration_message.py]: Hi Bonifase, how can I delete an alembic version upgrade @Bonifase

File: api/question/models.py [hunk 2]
```diff
@@ -13,3 +13,4 @@ class Question(Base, Utility):
     end_date = Column(String, nullable=False)
     total_views = Column(Integer, default=0)
     response = relationship('Response', cascade="all, delete-orphan")
+    is_active = Column(Boolean, nullable=True)
```
[the remark this answers is not in the document]
Comment (michael-basweti) [reply api/question/models.py]: noted, will change that
@caveinn

**Comment by joshuaocero** [comment]
This branch has potential merge conflicts. Please resolve them.

**Pull request merged**